doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -3,32 +3,118 @@
 Nixpkgs provides a variety of wrapper functions that help build commonly useful derivations.
 Like [`stdenv.mkDerivation`](#sec-using-stdenv), each of these build helpers creates a derivation, but the arguments passed are different (usually simpler) from those required by `stdenv.mkDerivation`.
 
-## `runCommand` {#trivial-builder-runCommand}
 
-`runCommand :: String -> AttrSet -> String -> Derivation`
+## `runCommandWith` {#trivial-builder-runCommandWith}
 
-The result of `runCommand name drvAttrs buildCommand` is a derivation that is built by running the specified shell commands.
+The function `runCommand` returns a derivation built using the specified command(s), in a specified environment.
 
-By default `runCommand` runs in a stdenv with no compiler environment, whereas [`runCommandCC`](#trivial-builder-runCommandCC) uses the default stdenv, `pkgs.stdenv`.
+It is a generalized version of `runCommand*`, whose behaviour is controlled via a single attribute set passed
+as the first argument, and allows specifying `stdenv` freely.
```

Comment:
```suggestion
as the first argument, and allows specifying `stdenv` freely.

The following `runCommand*` variants exist: 
- [runCommand](#trivial-builder-runCommand})
- (provide cross references here)
- ...
```

Reply:
We should do cross references, so it is easier to go back and forth between the functions.
With that i mean links that go in both directions.
runCommandWith <-> runCommand